releasenotes/notes/tphi-757e15fcb24219f9.yaml

```
@@ -0,0 +1,16 @@
+---
+
+features:
+  - |
+    New experiment Tphi. This is a composite experiment with sub-experiments
+    T1 and T2Ramsey.New classes:
```

```suggestion
    T1 and T2Ramsey. New classes:
```
I suggest to explain what's the new experiment about. The release notes are exposed to users and have to be informative for them.

---

Added a short description of the experiment.